### ISSUE TITLE
Ads: Ads are not removed from tab order using Talkback

### ISSUE THREAD

**Issue by tochwill** [bbc/simorgh#7755]
**Describe the bug**
The accessibility swarm highlighted a P2 bug, where not all links within canonical ads are removed from the tab order.

The ad iframe itself has tabindex="-1", and this means that when tabbing through the page using most browsers and assistive technology, the whole ad is skipped.

However, using Talkback with Chrome on Android - when swiping through links, the ad unit is selected in the tab order for both Leaderboards and MPUs. It selects just the ad unit itself and not the advertisement label.

We think this may be because the link itself has not been removed from the tab order.

If this is indeed the case, raise this with GNL to see how this can be resolved. Does this also happen on PAL pages with ads on talkback?

**To Reproduce**
Steps to reproduce the behaviour:

1. https://www.test.bbc.com/afrique from outside the UK using talkback for android
2. Tab through the page
4. See that the ad is selected by Talkback.

**Expected behaviour**
The ad slot should be completely ignored by Talkback (and other assistive technologies), and not accessible when tabbing through the page.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by rhenshaw56** [comment]
Blocked pending when we can use browserstck to reproduce this.

**Comment by greenc05** [comment]
@rhenshaw56 why is browserstack needed for this?

**Comment by pharingee** [comment]
Tested on TalkBack Android and unable to reproduce in Accra Ghana.

Device Information:
Model - Redmi Note 9S
Android Version - 10
MIUI  Versionn - 11.0.4

**Comment by joshcoventry** [comment]
For some reason there is a difference between Live and Test:

Live:
TalkBack skips ads on default navigation
Talkback skips ads on link navigation
tabindex=-1 present on PUBLICITES link
No tabindex attribute on iframe
No tabindex attribute on link inside iframe

Test:
TalkBack skips ads on default navigation
Talkback skips ads on link navigation
tabindex=-1 present on PUBLICITES link
tabindex=-1 present on iframe
No tabindex attribute on link inside iframe

We will need to investigate why the tabindex is not present on the iframe on Live.

**Comment by HarveyPeachey** [comment]
I've tried to replicate the bug by using a proxy to spoof my location. Testing Talkback on the Google pixel 5 I couldn't replicate the bug.

I've also checked using chrome dev tools to see why the `tabindex=-1` isn't present on the iframe on live, but when I checked I could see that `tabindex=-1` is present on the iframe.

**Comment by joshcoventry** [comment]
Closing as we cannot reproduce and the attribute has been seen on live.
